include/seqan3/alphabet/aminoacid/aa20.hpp

```diff
@@ -72,12 +72,12 @@ class aa20 : public aminoacid_base<aa20, 20>
     /*!\name Constructors, destructor and assignment
      * \{
      */
-    constexpr aa20() : base_t{} {}
-    constexpr aa20(aa20 const &) = default;
-    constexpr aa20(aa20 &&) = default;
-    constexpr aa20 & operator=(aa20 const &) = default;
-    constexpr aa20 & operator=(aa20 &&) = default;
-    ~aa20() = default;
+    constexpr aa20() : base_t{} {}                      //!< Empty
```

Comment:
This was a mistake originally also. Can you explicitly default the default constructor here and then document it as defaulted, too?

[If anything fails because of this, you can revert it again and I will take care of it. But it should work and if you are touching the lines anyway...]

Reply:
Can you also add the `noexcept` keyword and align the keywords? And don't forget the period at the end :nail_care: 

e.g
```cpp
    constexpr aa20()                         noexcept = default; //!< Defaulted.
    constexpr aa20(aa20 const &)             noexcept = default; //!< Defaulted.
    constexpr aa20(aa20 &&)                  noexcept = default; //!< Defaulted.
    constexpr aa20 & operator=(aa20 const &) noexcept = default; //!< Defaulted.
    constexpr aa20 & operator=(aa20 &&)      noexcept = default; //!< Defaulted.
    ~aa20()                                  noexcept = default; //!< Defaulted.
```

Thanks!

Reply:
But in the other doxygen fixes, I didn't add a period in the end... Do you want me to add them also to the already merged options as well?

Reply:
Hm, I am not sure, because I wasn't there when this was discussed in escala. @smehringer did you not discuss to add a `.` ?

In any case, you don't need to change things already changed, it would just apply to the new changes.

Reply:
I think discussing this takes too much time :sleepy: 
I really don't care about this dot. It makes this super important documentation one char longer... which maybe forces us to use the //\brief` version in probably 1% of the cases which is why I would allow "no dot".

Reply:
Hm, well, I also think we shouldn't discuss too long. I propose to have the dot, because everything else has it, but not change code that was already changed.

Reply:
I agree with @h-2. All documented code (even single word sentences) should end with a dot. The only exceptions are `\name` expressions which are captions.

Reply:
fine

Reply:
I got errors, adding the noexcept, so I didn't.